src/electron/auto-update/auto-updater-client.ts

```diff
@@ -1,8 +1,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import log from 'electron-log';
 
 export class AutoUpdaterClient {
-    constructor(private readonly autoUpdater) {}
+    constructor(private readonly autoUpdater) {
+        log.transports.file.level = 'info';
```

Comment:
In general, we try to avoid having global dependencies, instead we favor passing any dependency through the constructor. In the end, the composition root is the responsible to get the right instance for each dependency.

Reply:
I alternated between these options for a while & have pushed a new in-between iteration given your comment
```
// in main.ts:
import log from 'electron-log';
log.transports.file.level = 'info';
autoUpdater.logger = log;
const electronAutoUpdateCheck = new AutoUpdaterClient(autoUpdater);
```
or
```
// in auto-updater-client.ts:
import log from 'electron-log';
log.transports.file.level = 'info';
autoUpdater.logger = log; // in constructor
```

Reply:
spoke with @dbjorge / @waabid and settled on the first option above for now. We will look into `main.ts` refactoring separately, including how much of the update behavior to move into `AutoUpdaterClient` & how much of `main.ts` should be tested explicitly.